language/German/strings.po

```diff
@@ -1225,40 +1208,38 @@ msgid ""
 "\n"
 "If this is your first time using %2$s, please see the config.txt file in the "
 "docs directory for information on configuring %2$s."
-msgstr ""
+msgstr "Keine Maschinen gefunden. Bitte überprüfen Sie den Pfad für die ROMs in Ini-Datei %1$s.ini.\n\nWenn dies Ihre erste Verwendung von  %2$s ist, konsultieren Sie bitte config.txt im docs Verzeichnis zur Konfiguration von %2$s."
 
```

Comment:
Auf die Idee war ich soooo stolz...

Formuliere er einen Alternativvorschlag und wir werden gnädigst eine
Anpassung in Betrachtung ziehen.

Schwebte Dir sowas wie
"

Wenn dies Ihre erste Verwendung von  %2$s ist, lesen Sie bitte
config.txt im docs Verzeichnis zur Konfiguration von %2$s."

vor?

Könnte ich auch mit leben...


Am 29. Februar 2016 um 00:50 schrieb Michael Zapf <notifications@github.com>
:

> In language/German/strings.po
> <https://github.com/mamedev/mame/pull/660#discussion_r54359453>:
>
> > @@ -1225,40 +1208,38 @@ msgid ""
> >  "\n"
> >  "If this is your first time using %2$s, please see the config.txt file in the "
> >  "docs directory for information on configuring %2$s."
> > -msgstr ""
> > +msgstr "Keine Maschinen gefunden. Bitte überprüfen Sie den Pfad für die ROMs in Ini-Datei %1$s.ini.\n\nWenn dies Ihre erste Verwendung von  %2$s ist, konsultieren Sie bitte config.txt im docs Verzeichnis zur Konfiguration von %2$s."
> >
>
> Also, ich lese gelegentlich mal in einer Datei oder schaue hinein, aber
> "konsultieren"? Klingt mir ein bisschen fürnehm (natürlich eine
> Geschmacksfrage).
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mamedev/mame/pull/660/files#r54359453>.
>